web-api/src/persistence/postgres/featureFlag/schema.ts

```diff
@@ -0,0 +1,18 @@
+import { Selectable, Insertable, Updateable } from 'kysely';
+
+const DEFAULT = {};
+
+export const featureFlagTableDefinition = {
+  name: DEFAULT as string,
+  value: DEFAULT as { current: any },
+};
+
+export type FeatureFlagTable = typeof featureFlagTableDefinition;
+
+export const DW_FEATURE_FLAG_COLUMNS = Object.keys(
+  featureFlagTableDefinition,
+) as Array<keyof FeatureFlagTable>;
+
+export type FeatureFlagKysely = Selectable<FeatureFlagTable>;
+export type NewFeatureFlagKysely = Insertable<FeatureFlagTable>;
+export type UpdateFeatureFlagKysely = Updateable<FeatureFlagTable>;
```

Comment:
Can we delete this? I can explain more details if you want, but our app as it stands should probably never use `Updateable`, and it looks like it is unused in this PR.